pages/linux/ss.md

```diff
@@ -2,26 +2,26 @@
 
 > Utility to investigate sockets.
 
-- Dump all TCP/UDP/RAW/UNIX sockets:
+- Show all TCP/UDP/RAW/UNIX sockets:
 
 `ss -a {{-t/-u/-w/-x}}`
```

Comment:
The ipv4/6 example I originally fixed by having `ss -{{4|6}}`. The dash can be inside the braces even, but I'd insist on having `|` for separators we've had a couple of times.